lib/blockchain/chaindb.js

```diff
@@ -1108,6 +1108,30 @@ class ChainDB {
     return coin.toCoin(prevout);
   }
 
+  /**
+   * Get all coins.
+   * @returns {Promise} - Returns {@link Coin}.
+   */
+
```

Comment:
If we ever wanted to add `scantxoutset`, this code path would be useful.
https://developer.bitcoin.org/reference/rpc/scantxoutset.html

I imagine a "filter" function passed into `getCoins` that returns a bool that would determine if the `coin` is added to `coins`. If no function is passed in, this filter function would always evaluate true. I think writing a basic parser for output descriptors would be pretty straight forward, although it would likely run into the the issue here: https://github.com/handshake-org/hs-client/pull/27